frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Create Application from git form
+    As a user, I want to deploy an application from git repo with devfile
+
+
+    Background:
+        Given user is at developer perspective
+        And user is at Add page
```

Comment:
You can remove this from here as it varies in the scenarios

Reply:
Updated